openmoc/options.py

```diff
@@ -169,15 +171,15 @@ def parseArguments(self):
                 num_azim += 'the number of azimuthal angles\n'
                 print(num_azim)
 
-                num_polar = '\t{: <35}'.format('-a, --num-polar=<6>')
+                num_polar = '\t{: <35}'.format('-r, --num-polar=<6>')
```

Comment:
-p feels more natural than -r for the number of polar angles.

Reply:
Agree. Fixed.